TcUnit-Runner/Program.cs

```diff
@@ -119,14 +122,16 @@ static void Main(string[] args)
             try
             {
                 vsInstance = new VisualStudioInstance(@VisualStudioSolutionFilePath, tcVersion);
-                vsInstance.Load();
+                var tcPinnedVersion = XmlUtilities.GetPinnedVersionXML(TwinCATProjectFilePath);
```

Comment:
possible to replace VAR's to the actual type?

Reply:
Yes
`bool tcPinnedVersion = XmlUtilities.GetPinnedVersionXML(TwinCATProjectFilePath);`

Reply:
Great!